### PR TITLE
feat(config): import Z-Wave Alliance Files batch 10

### PR DESCRIPTION
**Pull request by blhoward2**
Batch 10 for volunteer review

### CODE REVIEW

**Comment by tradergene** [comment]
I'll tackle this one.

**Comment by tradergene** [comment]
@blhoward2 please review when you get a chance.

**Review by tradergene — CHANGES_REQUESTED** [review]
Accidental deletion of .gitignore

File: packages/config/config/devices/0x0212/gda-a2e6b-k0.json [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Silent Mode On/Off",
+			"description": "",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0212/gda-a2e6b-k0.json]: remove

File: packages/config/config/devices/0x0212/gda-a2e6b-k0.json [hunk 2]
```diff
@@ -0,0 +1,88 @@
+// GATEMAN GDA-A2E6B-K0
+// A20-SH
+{
+	"manufacturer": "GATEMAN",
+	"manufacturerId": "0x0212",
+	"label": "GDA-A2E6B-K0",
+	"description": "A20-SH",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0212/gda-a2e6b-k0.json]: Should that be part of the label? If not, either remove it or find a real description :)

File: packages/config/config/devices/0x0212/gda-a2e6b-k0.json [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+				},
+				{
+					"label": "Enable auto relock",
+					"value": 255
+				}
+			]
+		},
+		"3": {
+			"label": "Relock Time",
+			"description": "This value from 10 to 180 seconds means the time for lock to retain the unlocked state before doing the auto relock",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0212/gda-a2e6b-k0.json]:
```suggestion
			"description": "How long the lock stays unlocked before automatically relocking.",
```

Comment (AlCalzone) [reply packages/config/config/devices/0x0212/gda-a2e6b-k0.json]: or something in this regard

File: packages/config/config/devices/0x0212/gda-a2e6b-k0.json [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+			"defaultValue": 30,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		}
+	},
+	"metadata": {
+		"inclusion": "① Touch Keypad with palm.\n② Enter User Code(Normal Mode only. Enter Master Code if operating Master Mode).\n③ Press 'I' button [Registration] from the inside of battery cover. \n④ Press 9, #, 1, # by sequence on the keypad(to Add).\n⑤ Wait until \"Ding dong dang\" melody play, then press 'I' button [Registration] from the inside of battery cover",
+		"exclusion": "① Touch Keypad with palm.\n② Enter User Code(Normal Mode only. Enter Master Code if operating Master Mode).\n③ Press 'I' button [Registration] from the inside of battery cover. \n④ Press 9, #, 3, # on the keypad by sequence (to Delete).\n⑤ Enter User Code(Normal Mode only. Enter Master Code if operating Master Mode), then press #.\n⑤ Wait until \"Ding dong dang\" melody play, then press 'I' button [Registration] from the inside of battery cover",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0212/gda-a2e6b-k0.json]: There are a bunch of special characters that shouldn't be here. Check whitespace and spellcheck please.

File: packages/config/config/devices/0x0212/gdm-m2d6d-k0.json [hunk 5]
```diff
@@ -0,0 +1,87 @@
+// GATEMAN GDM-M2D6D-K0
+// A200-CH
+{
+	"manufacturer": "GATEMAN",
+	"manufacturerId": "0x0212",
+	"label": "GDM-M2D6D-K0",
+	"description": "A200-CH",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0212/gdm-m2d6d-k0.json]: see previous file

File: packages/config/config/devices/0x0212/gdm-m2d6d-k0.json [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+				},
+				{
+					"label": "Enable auto relock",
+					"value": 255
+				}
+			]
+		},
+		"3": {
+			"label": "Relock Time",
+			"description": "",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0212/gdm-m2d6d-k0.json]: see previous file

File: packages/config/config/devices/0x0212/gdm-m2d6d-k0.json [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+		"exclusion": "To exclude:\n1. Touch keypad with palm\n2. Enter User code(Normal Mode only. Enter Master code if operating Master Mode.)\n3. Press 'R' button [Registration] from the inside of battery cover.\n4. Press 9,#,3,# on the keypad (by sequence)\n5. Wait until 'Ding dong dang' melody play than press 'R' button [Registration]",
+		"reset": "To reset to factory defaults:\n ※ Use this procedure only in the event that the network primary controller is missing or otherwise inoperable. (User code or master code remains the same. Only network settings reset to factory defaults.) \n1. Touch keypad with palm\n2. Enter User code(Normal Mode only. Enter Master code if operating Master Mode.)\n3. Press 'R' button [Registration] from the inside of battery cover.\n4. Press 9,#,9,# on the keypad (by sequence)\n5. Wait until 'Ding dong dang' melody play than press 'R' button [Registration]",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0212/gdm-m2d6d-k0.json]: See previous file

File: packages/config/config/devices/0x021f/dmex1.json [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Night Light Off Threshold",
+			"description": "",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x021f/dmex1.json]: remove

Comment (AlCalzone) [reply packages/config/config/devices/0x021f/dmex1.json]: Or is there a better description?

Comment (AlCalzone) [reply packages/config/config/devices/0x021f/dmex1.json]: same goes for all other empty descriptions.

File: packages/config/config/devices/0x022c/osr-01.json [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Battery Report Period",
+			"description": "Value x 15 minute increments",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x022c/osr-01.json]: if a setting of 1 means 15 minutes, the unit should be `15 minutes` instead.

File: packages/config/config/devices/0x022c/osr-01.json [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		}
+	},
+	"metadata": {
+		"inclusion": "a. Press & hold Inclusion/Exclusion button for 5 seconds when Open Sensor is\n exclusion status.\nb. LED of Open sensor will be on 5 seconds later.\nc. Open sensor will try Inclusion for 30sec when button is released.\nd. LED will blink 3 times if Inclusion is successful.",
+		"exclusion": "a. Receiver will stay exclusion waiting mode.\nb. Press & hold Inclusion/Exclusion button for 5 seconds when Open Sensor is\n Inclusion status.\nc. LED of Open sensor will be on 5 seconds later.\nd. Open sensor will try Exclusion when button is released.\ne. LED will blink 3 times if Exclusion is successful.",
+		"reset": "a. Press & hold Inclusion/Exclusion button for 10sec.\nb. LED will blink while holding the button.\nc. Factory reset will be done in 5sec after releasing button and LED will blink 5 times.\n\nPlease use this procedure only when the network primary controller is missing or otherwise inoperable",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x022c/osr-01.json]: check and cleanup whitespace and line breaks

File: packages/config/config/devices/0x022c/osr-02.json [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Battery Report Period",
+			"description": "Value x 15 minute increments",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x022c/osr-02.json]: see previous file

File: packages/config/config/devices/0x022c/osr-02.json [hunk 12]
```diff
@@ -0,0 +1,39 @@
...
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		}
+	},
+	"metadata": {
+		"inclusion": "a. Press & hold Inclusion/Exclusion button for 5 seconds when Door Sensor is exclusion status.\nb. LED of Door sensor will be on 5 seconds later.\nc. Door sensor will try Inclusion for 30sec when button is released.\nd. LED will blink 3 times with 250ms interval if Inclusion is successful",
+		"exclusion": "a. Receiver will stay exclusion waiting mode.\nb. Press & hold Inclusion/Exclusion button for 5 seconds when Door Sensor is Inclusion status.\nc. LED of Door sensor will be on 5 seconds later.\nd. Door sensor will try Exclusion when button is released.\ne. LED will blink 3 times with 250ms interval if Exclusion is successful",
+		"reset": "a. Press & hold Inclusion/Exclusion button for 10sec.\nb. LED will blink while holding the button.\nc. Factory reset will be done in 5sec after releasing button and LED will blink 5 times with 100ms interval.\n* Please use this procedure only when the network primary controller is missing or otherwise inoperable",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x022c/osr-02.json]: see previous file

File: packages/config/config/devices/0x022c/sa08.json [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Battery Report Period",
+			"description": "Value x 30 minute increments",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x022c/sa08.json]: see previous file

File: packages/config/config/devices/0x022c/sa08.json [hunk 14]
```diff
@@ -0,0 +1,40 @@
...
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		}
+	},
+	"metadata": {
+		"inclusion": "a. Press & hold Inclusion/Exclusion button for 5 seconds when Open Sensor is exclusion status.\nb. LED of Open sensor will be on 5 seconds later.\nc. Open sensor will try Inclusion for 30sec when button is released.\nd. LED will blink 3 times if Inclusion is successful.",
+		"exclusion": "a. Receiver will stay exclusion waiting mode.\nb. Press & hold Inclusion/Exclusion button for 5 seconds when Open Sensor is Inclusion status.\nc. LED of Open sensor will be on 5 seconds later.\nd. Open sensor will try Exclusion when button is released.\ne. LED will blink 3 times if Exclusion is successful.",
+		"reset": "a. Press & hold Inclusion/Exclusion button for 10sec.\nb. LED will blink while holding the button.\nc. Factory reset will be done in 5sec after releasing button and LED will blink 5 times.\nPlease use this procedure only when the network primary controller is missing or otherwise inoperable.",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x022c/sa08.json]: see previous file

File: packages/config/config/devices/0x0249/psr07.json [hunk 15]
```diff
@@ -0,0 +1,85 @@
...
+			"minValue": 0,
+			"maxValue": 99,
+			"defaultValue": 99,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"10": {
+			"label": "Auto Report Battery Time",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0249/psr07.json:43]: unit?

File: packages/config/config/devices/0x0249/psr07.json [hunk 16]
```diff
@@ -0,0 +1,85 @@
...
+			"minValue": 1,
+			"maxValue": 127,
+			"defaultValue": 12,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"25": {
+			"label": "Percentage Set Method",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0249/psr07.json]: The options don't make sense for this label IMO. Maybe there's a better one?

File: packages/config/config/devices/0x024c/ht-a10.json [hunk 17]
```diff
@@ -0,0 +1,51 @@
...
+			"zwaveAllianceId": 1687
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Timer Setting",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024c/ht-a10.json:21]: unit?

File: packages/config/config/devices/0x024c/ht-a10.json [hunk 18]
```diff
@@ -0,0 +1,51 @@
...
+			"minValue": 0,
+			"maxValue": 54,
+			"defaultValue": 1,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"2": {
+			"label": "Remained Time",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024c/ht-a10.json:33]: unit?

File: packages/config/config/devices/0x024d/dn3g6ja062.json [hunk 19]
```diff
@@ -0,0 +1,70 @@
...
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "LED indicator is always off",
+					"value": 0
+				},
+				{
+					"label": "LED indicator lights 1 second after thumb turn status close to open",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024d/dn3g6ja062.json]:
```suggestion
					"label": "LED indicator lights 1 second after opening",
```

File: packages/config/config/devices/0x024d/dn3g6ja062.json [hunk 20]
```diff
@@ -0,0 +1,70 @@
...
+					"value": 1
+				},
+				{
+					"label": "LED indicator lights 1 second after thumb turn status open to close",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024d/dn3g6ja062.json]:
```suggestion
					"label": "LED indicator lights 1 second after closing",
```

File: packages/config/config/devices/0x024d/dn3g6ja062.json [hunk 21]
```diff
@@ -0,0 +1,70 @@
...
+					"value": 2
+				},
+				{
+					"label": "LED indicator lights 1 second after changed thumb turn status",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024d/dn3g6ja062.json]:
```suggestion
					"label": "LED indicator lights 1 second after opening and closing",
```

File: packages/config/config/devices/0x024f/ad1-10v.json [hunk 22]
```diff
@@ -0,0 +1,158 @@
...
+					"label": "1-Pole 16A Relay",
+					"value": 19
+				}
+			]
+		},
+		"10": {
+			"label": "Dimming Speed",
+			"description": "",
+			"valueSize": 2,
+			"unit": ".01 seconds",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024f/ad1-10v.json]:
```suggestion
			"unit": "0.01 seconds",
```

File: packages/config/config/devices/0x024f/ad1-10v.json [hunk 23]
```diff
@@ -0,0 +1,158 @@
...
+			"maxValue": 32767,
+			"defaultValue": 300,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"11": {
+			"label": "Dimmer Mode",
+			"description": "Not applicable",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024f/ad1-10v.json]: remove

File: packages/config/config/devices/0x024f/am.json [hunk 24]
```diff
@@ -0,0 +1,158 @@
...
+					"label": "1-Pole 16A Relay",
+					"value": 19
+				}
+			]
+		},
+		"10": {
+			"label": "Dimming Speed",
+			"description": "",
+			"valueSize": 2,
+			"unit": ".01 seconds",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024f/am.json]:
```suggestion
			"unit": "0.01 seconds",
```

File: packages/config/config/devices/0x024f/am.json [hunk 25]
```diff
@@ -0,0 +1,158 @@
...
+			"maxValue": 32767,
+			"defaultValue": 300,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"11": {
+			"label": "Dimmer Mode",
+			"description": "Not applicable",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x024f/am.json]: see previous file

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x0212/gda-a2e6b-k0.json (hunk 3 above)
Comment (tradergene) [reply packages/config/config/devices/0x0212/gda-a2e6b-k0.json:70]: Fixed.

File: packages/config/config/devices/0x0212/gda-a2e6b-k0.json (hunk 4 above)
Comment (tradergene) [reply packages/config/config/devices/0x0212/gda-a2e6b-k0.json:84]: I think I removed all special characters and missing whitespace. Not seeing spelling issues. Left the escape characters, which I assume you want to keep? If not let me know and I'll remove the newlines and escaped quotes.

Comment (AlCalzone) [reply packages/config/config/devices/0x0212/gda-a2e6b-k0.json]: > newlines and escaped quotes

need to stay. I meant the ①② etc.

File: packages/config/config/devices/0x0212/gda-a2e6b-k0.json (hunk 1 above)
Comment (tradergene) [reply packages/config/config/devices/0x0212/gda-a2e6b-k0.json]: You mean remove empty descriptions?

Comment (AlCalzone) [reply packages/config/config/devices/0x0212/gda-a2e6b-k0.json]: Yep. See https://zwave-js.github.io/node-zwave-js/#/development/config-files?id=paraminformation

File: packages/config/config/devices/0x0249/psr07.json (hunk 15 above)
Comment (tradergene) [reply packages/config/config/devices/0x0249/psr07.json:43]: There's no unit reference in the manual nor on the alliance page but the default is 12.  Based on that, I'm going with hours.

File: packages/config/config/devices/0x0249/psr07.json (hunk 16 above)
Comment (tradergene) [reply packages/config/config/devices/0x0249/psr07.json]: I agree, but it's a device I've never seen before.  I changed this to "Dimmer Set Method" which seems to be more accurate per the manual.

File: packages/config/config/devices/0x0249/psr07.json (hunk 15 above)
Comment (tradergene) [reply packages/config/config/devices/0x0249/psr07.json:43]: Actually, looks like the default is 6 hours, per another part of the manual, so I'm going with "0.5 hours"

**Comment by tradergene** [comment]
@AlCalzone I don't seem to have permission to request review, but I made the changes you requested, please take a look.

**Pull request closed without merging**